docs/install/index.md

```diff
@@ -459,7 +459,16 @@ $ make -j $(nproc) USE_OPENCV=1 USE_BLAS=openblas
 $ sudo apt-get install -y python-dev python-setuptools python-pip libgfortran3
 ```
 
-**Step 2** Install the MXNet Python binding.
+**Step 2** Build cython modules (optional).
```

Comment:
Can we please add instructions for Mac and Windows users?

Reply:
Unfortunately I'm not familiar with Windows/Mac and also don't have environments to test. I think this needs help from others.

Reply:
@sandeep-krishnamurthy maybe we can start with Linux only, and leverage other's help to support Mac and Windows?

Reply:
Yes we can iteratively update.